client/src/app/pages/controls/business-services/components/business-service-form.tsx

```diff
@@ -94,6 +92,59 @@ export const BusinessServiceForm: React.FC<BusinessServiceFormProps> = ({
     mode: "onChange",
   });
 
+  const onCreateBusinessServiceSuccess = (
+    response: AxiosResponse<BusinessService>
+  ) => {
+    pushNotification({
+      title: t("toastr.success.createWhat", {
+        type: t("terms.businessService"),
+        what: response.data.name,
+      }),
+      variant: "success",
+    });
+    onClose();
+  };
+
+  const onUpdateBusinessServiceSuccess = (
```

Comment:
We should be using a combined success handler here like we are for applications. Or at least we should agree on a style and be consistent across slices of the app.

Reply:
@ibolton336,

This seems to be out of scope, please https://github.com/konveyor/tackle2-ui/issues/1214 to following-up.

Reply:
Thx. I read the issue but I still don't see why the application handler is a special case. Maybe it would just be easier to move the on create / on update success handlers to their own functions for the applications? Just for consistency.

Reply:
@ibolton336, 
you're right, the Application isn't a special case, I confused it with the MigrationTargets where we have to pass the reference to scroll down the page.
So revisiting this, we want to be consistent and have separate handlers, I'm changing the handlers in https://github.com/konveyor/tackle2-ui/pull/1207.

We might eventually revisit that, consistently of course, or not once we decide to pass along the object name to the save/edit.